skimage/measure/_find_contours_cy.pyx

```diff
@@ -25,7 +25,7 @@ def iterate_and_store(double[:, :] array,
     nonzero, high-values pixels are considered to be face+vertex
     connected into objects; otherwise low-valued pixels are.
 
-    Positions where the boolean array mask is False are considered
+    Positions where the boolean array ``mask`` is ``False`` are considered
```

Comment:
Do you deliberately choose to format variables as literals in docstrings? The [convention (section Parameters)](https://numpydoc.readthedocs.io/en/latest/format.html#sections) I know is to emphasize with a single surrounding \`, e.g. \`mask\`.

Reply:
Not to block this PR. I'm just curious.

Reply:
You're correct! I've gotten into the habit of double backticks as they are always safe. Fix coming...

Reply:
Fixed in cde8c3137